components/Section.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export default function Wrapper({ children }) {
+  return <StudioGeneratorSection>{children}</StudioGeneratorSection>;
+}
```

Comment:
you can export the component "StudioGeneratorSection" directly. You don't need to create a second Component here called Wrapper. You can do that like this:
```js

const StudioGeneratorSection = styled.div`
  ...styling
`
export default StudioGeneratorSection
```
Keep in mind that a default export can be imported under any name you like, they can be "renamed when imported" so to speak. So it doesn't matter which variable name you choose here, when you import it you can choose a different name.